docker-compose.yml

```diff
@@ -1,18 +1,25 @@
-version: "3.8"
+version: "3"
+
 services:
-  redis:
-    image: redis:6-alpine
-    restart: unless-stopped
-    volumes:
-      - redis-data:/var/lib/redis
+
   nitter:
     image: zedeus/nitter:latest
-    restart: unless-stopped
-    depends_on:
-      - redis
+    container_name: nitter
     ports:
-      - "8080:8080"
+      - "127.0.0.1:8080:8080" # Replace with "8080:8080" if you don't use a reverse proxy
     volumes:
       - ./nitter.conf:/src/nitter.conf
+    depends_on:
+      - nitter-redis
+    restart: unless-stopped
+
+  nitter-redis:
```

Comment:
Why change the name of the service? Wouldn't this break existing installs if they decided to `git pull`?

Reply:
>Rename the redis container so that it's nitter specific

It make it so that another software won't use it.

It won't be a problem, since currently it did "nothing" since it was all in RAM.